src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
>Since config file is user editable it is a really good idea to store enumerations by name 

Now I'm not so sure. As a developer, it's sure convenient.
As a user, how is one going to know the only valid values are `FixedSlotsChoker, RateBasedChoker` without source code at hand?
On the other way, if it's represented as an `int`, at least user can try putting numbers in it.